openbb_platform/extensions/stocks/integration/test_stocks_api.py

```diff
@@ -1000,3 +1000,61 @@ def test_stocks_disc_active(params, headers):
     result = requests.get(url, headers=headers, timeout=10)
     assert isinstance(result, requests.Response)
     assert result.status_code == 200
+
+
+@pytest.mark.parametrize(
+    "params",
+    [
+        ({"symbol": "AAPL", "provider": "sec"}),
```

Comment:
will this not fail?

Reply:
If we bump the timeout to 30 it will not

![Screenshot 2023-10-31 at 14 50 41](https://github.com/OpenBB-finance/OpenBBTerminal/assets/74266147/ebd2634b-dd54-4b97-913d-1315f2ffc524)